docs/source/features/passes/pytorch.md

```diff
@@ -183,6 +183,24 @@ This pass only supports Hugging Face transformers PyTorch models. Please refer t
 }
 ```
 
+## SliceGPT
+`SliceGPT` is post-training sparsification scheme that makes transformer networks smaller by applying orthogonal transformations to each transformer layer that reduces the model size by slicing off the least-significant rows and columns of the weight matrices. This results in speedups and a reduced memory footprint.
+
+Please refer to the original [paper](https://arxiv.org/abs/2401.15024) for more details on the algorithm and expected results for different models, sparsities and datasets.
+
+This pass only supports Hugging Face transformers PyTorch models. Please refer to [SparseGPT](sparsegpt) for more details on the types of transformers models supported.
```

Comment:
Why do you need to refer to SparseGPT here? What if the list of supported model differ from each other?

Reply:
That should have been SliceGPT .. typo error. Fixing it.

Reply:
Fixed.